Expecto.Tests/Tests.fs

```diff
@@ -184,8 +184,8 @@ let timeouts =
           match testFromType Dummy.thisModuleType.Value with
           | Some (TestList (
                       Seq.Two (
-                          TestLabel("test B", TestList _),
-                          TestLabel("test A", TestList _)))) -> ()
+                          TestLabel("test B", TestList (_, Normal), Normal),
```

Comment:
Removing focus state from *TestLabel* would break the propagation of *FocusState* for a structure like:
```fsharp
ftestList "all should run" [ TestLabel "b" (test "should run as parent is focused" { 1 ==? 2 })]
ftest "will run" { 1 ==? 1 }
```
The workaround will make the code complex and harder to follow (when we need to propagate the state, we need to decompose childs until a child with state is found and then translate (combine) the states.